djangoproject/templates/includes/footer.html

```diff
@@ -48,6 +48,7 @@ <h2>Follow Us</h2>
           <ul>
             <li><a href="https://github.com/django">GitHub</a></li>
             <li><a href="https://twitter.com/djangoproject">Twitter</a></li>
+            <li><a href="https://fosstodon.org/@django">Fediverse (Mastodon)</a></li>
```

Comment:
If this is "the" Django account, I'd suggest adding [link verification](https://docs.joinmastodon.org/user/profile/#verification) with this change, so it's reflected back on the account profile. 

The change to apply this would be the following:
```suggestion
            <li><a href="https://fosstodon.org/@django" rel="me">Fediverse (Mastodon)</a></li>
```